src/components/searchStudent/SearchStudent.tsx

```diff
@@ -0,0 +1,58 @@
+
+import './SearchStudent.css';
+import Navbar from '../navbar/Navbar';
+import SearchBar from '../SearchBar';
+
+const SearchStudent = () => {
+  return (
+    <div>
+        <Navbar />
```

Comment:
I think you should use React router <Outlet/>
Create a Layout component, put 
<Navbar/>
<Outlet/> 
and router will replace the Outlet with your component instead of having to add the Navbar manually each time